content/sensu-go/6.2/observability-pipeline/observe-schedule/metrics.md

```diff
@@ -0,0 +1,365 @@
+---
+title: "Metrics reference"
+linkTitle: "Metrics Reference"
+reference_title: "Metrics"
+type: "reference"
+description: "Use checks to collect and process service metrics for your infrastructure with the Sensu observability pipeline. Read this reference doc to learn about Sensu Go's first-class support for collecting and processing metrics."
+weight: 33
+version: "6.2"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.2:
+    parent: observe-schedule
+---
+
+Sensu Go offers first-class built-in support for collecting and processing metrics for your entire infrastructure.
+
+In Sensu, metrics are events that contain metrics data.
+Sensu metrics can be metrics-only events created with data from the [Sensu StatsD listener][2] or [Prometheus][7].
+They can also be status and metrics events from [check output metric extraction][4].
+
```

Comment:
Maybe we should add a note here to explain that the metrics reference does not describe the same functionality as the [metrics API](https://docs.sensu.io/sensu-go/latest/api/metrics/). I'm not sure if that's a common misunderstanding, but most of our other references coordinate with an API doc so it might be worth mentioning.

Reply:
Added